styles/search-input.scss

```diff
@@ -1,8 +1,14 @@
+@import "mixins";
+
 .c-search-input {
 	border-radius: 3px;
 	position: relative;
 	z-index: 0;
-	overflow: hidden;
+	flex-grow: 1;
```

Comment:
this `flex-grow` should be applied everywhere for the search input on the project, right?